tests/model/generated/object/test_notification_url.py

```diff
@@ -0,0 +1,217 @@
+    # Getter string constants
+    _GETTER_PAYMENT = 'Payment'
+    _GETTER_BUNQ_ME_TAB = 'BunqMeTab'
+    _GETTER_CHAT_MESSAGE_ANNOUNCEMENT = 'ChatMessageAnnouncement'
+    _GETTER_DRAFT_PAYMENT = 'DraftPayment'
+    _GETTER_MASTER_CARD_ACTION = 'MasterCardAction'
+    _GETTER_MONETARY_ACCOUNT_BANK = 'MonetaryAccountBank'
+    _GETTER_PAYMENT_BATCH = 'PaymentBatch'
+    _GETTER_REQUEST_INQUIRY = 'RequestInquiry'
+    _GETTER_REQUEST_RESPONSE = 'RequestResponse'
+    _GETTER_SCHEDULE_PAYMENT = 'ScheduledPayment'
+    _GETTER_SCHEDULE_INSTANCE = 'ScheduledInstance'
+    _GETTER_SHARE_INVITE_BANK_INQUIRY = 'ShareInviteBankInquiry'
+    _GETTER_SHARE_INVITE_BANK_RESPONSE = 'ShareInviteBankResponse'
+
+    # Model json paths constants.
+    BASE_PATH_JSON_MODEL = '../../../assets/NotficationUrlJsons'
+    JSON_PATH_MUTATION_MODEL = BASE_PATH_JSON_MODEL + '/Mutation.json'
+    JSON_PATH_BUNQ_ME_TAB_MODEL = BASE_PATH_JSON_MODEL + '/BunqMeTab.json'
+    JSON_PATH_CHAT_MESSAGE_ANNOUNCEMENT_MODEL = \
+        BASE_PATH_JSON_MODEL + '/ChatMessageAnnouncement.json'
+    JSON_PATH_DRAFT_PAYMENT_MODEL = BASE_PATH_JSON_MODEL + '/DraftPayment.json'
+    JSON_PATH_MASTER_CARD_ACTION_MODEL = \
+        BASE_PATH_JSON_MODEL + '/MasterCardAction.json'
+    JSON_PATH_MONETARY_ACCOUNT_BANK_MODEL = \
+        BASE_PATH_JSON_MODEL + '/MonetaryAccountBank.json'
+    JSON_PATH_PAYMENT_BATCH_MODEL = \
+        BASE_PATH_JSON_MODEL + '/PaymentBatch.json'
+    JSON_PATH_REQUEST_INQUIRY_MODEL = \
+        BASE_PATH_JSON_MODEL + '/RequestInquiry.json'
+    JSON_PATH_REQUEST_RESPONSE_MODEL = \
+        BASE_PATH_JSON_MODEL + '/RequestResponse.json'
+    JSON_PATH_SCHEDULE_PAYMENT_MODEL = \
+        BASE_PATH_JSON_MODEL + '/ScheduledPayment.json'
+    JSON_PATH_SCHEDULE_INSTANCE_MODEL = \
+        BASE_PATH_JSON_MODEL + '/ScheduledInstance.json'
+    JSON_PATH_SHARE_INVITE_BANK_INQUIRY_MODEL = \
+        BASE_PATH_JSON_MODEL + '/ShareInviteBankInquiry.json'
+    JSON_PATH_SHARE_INVITE_BANK_RESPONSE_MODEL = \
+        BASE_PATH_JSON_MODEL + '/ShareInviteBankResponse.json'
+
+    # Model root key.
+    _KEY_NOTIFICATION_URL_MODEL = 'NotificationUrl'
+
+    def execute_notification_url_test(self,
+                                      file_path,
+                                      class_name,
+                                      getter_name):
+        """
+        :type file_path: str
+        :type class_name: str
+        :type getter_name: str
+
+        :return: None
+        """
+
+        notification_url = self.getNotificationUrl(file_path)
+        self.assertIsNotNone(notification_url)
+        self.assertIsNotNone(notification_url.object_)
+
+        expected_model = getattr(notification_url.object_, getter_name)
+        referenced_model = notification_url.object_.get_referenced_object()
+
+        self.assertIsNotNone(expected_model)
+        self.assertIsNotNone(referenced_model)
+        self.assertTrue(
+            self.assertInstanceOfReferencedObject(
```

Comment:
this should not be called "assert..." but just "is...", because it is not throwing any errors!